### DISCUSSION TITLE
Remove data when deleting the post

### DISCUSSION THREAD

**Discussion by dcexc** [eduardo-marcolino/acf-fields-in-custom-table#10] [General]
When using the ACF fields to create a post, when you delete the post the data doesn't remove from the custom table. Any chance of implementing some sort of cross link so it removes the data when the post is deleted?

**Comment by eduardo-marcolino** [comment]
there is a hook for post delete already: https://github.com/eduardo-marcolino/acf-fields-in-custom-table/blob/f3f02da4aa0cf797516861e4605550834ee48536/acf-fields-in-custom-table.php#L78

**Comment by dcexc** [comment]
how do i use it? there no button for it or anything?

**Reply by eduardo-marcolino** [reply]
Nothing needs to be done. Once you delete the record **permanently** the record will be removed from the custom table.
The table it self will not be deleted.
